apps/src/templates/sectionsRefresh/coteacherSettings/AddCoteacher.jsx

```diff
@@ -195,6 +196,10 @@ export default function AddCoteacher({
   const isMaxCoteachers = numCoteachers >= 5;
 
   const getErrorOrCount = () => {
+    if (loginType === 'ltiV1') {
```

Comment:
Do you know why the value here is "ltiV1"? I was going to suggest using the value from `SectionLoginType` in `SharedConstants`, but that value is "lti_v1". Everywhere else I've seen it, it's snake case rather than camel case.

Reply:
Oh dang, does that actually camelize the _values_? Makes me wonder if that's a bug and the intended behavior was to camelize the keys of the object before stringifying it.

Anyway, I guess this seems to fit with the way the page works. It seems to use the old sections controller instead of the "v1" controller, so we get the camelized section instead of the result from the `summarize` method that is used in other parts of the app.